### PR TITLE
Updating AdMob Adapter to use Unified native Ad and MediaView.

### PR DESCRIPTION
**Pull request by maninomula**
* Added Unified Native Ad and removed deprecated native ad types.
* Added MediaView instance.

### CODE REVIEW

**Comment by rsatrasala** [comment]
Thank you @maninomula for the PR. We will look into this shortly.

**Comment by chauduyphanvu1** [comment]
@maninomula, do you have a test video creative for us to test the MediaView?

**Comment by maninomula** [comment]
@chauduyphanvu1, You can use the [Google Test Ads](https://developers.google.com/admob/ios/test-ads#sample_ad_units) for Video Ads.

File: AdMob/MPGoogleAdMobNativeAdAdapter.m [hunk 1]
```diff
@@ -105,26 +60,35 @@ - (instancetype)initWithAdMobNativeAppInstallAd:(GADNativeAppInstallAd *)adMobNa
...
+- (void)nativeAdDidRecordClick:(GADUnifiedNativeAd *)nativeAd {
   // Sending click to MoPub SDK.
   [self.delegate nativeAdDidClick:self];
 }
 
 #pragma mark - <MPNativeAdAdapter>
 
 - (UIView *)privacyInformationIconView {
-  return _adChoicesView;
+  return self.adMobUnifiedNativeAdView.adChoicesView;
```
Comment (rsatrasala) [inline AdMob/MPGoogleAdMobNativeAdAdapter.m]: Does this view require initialization? Its cleaned up above 
`_adChoicesView = [[GADAdChoicesView alloc] initWithFrame:CGRectMake(0, 0, 20, 20)];`

Comment (rsatrasala) [reply AdMob/MPGoogleAdMobNativeAdAdapter.m]: ok the initialization is happening elsewhere. Consider this resolved.

**Pull request merged**